modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/internal/util/PingbackMethodImpl.java

```diff
@@ -388,6 +394,30 @@ protected Response validateSource() throws Exception {
 			SOURCE_URI_INVALID, "Unable to find target URI in source");
 	}
 
+	private boolean _isSourceURILocalNetwork() {
+		try {
+			URI uri = new URI(_sourceURI);
+
+			InetAddress ipaddress = InetAddress.getByName(uri.getHost());
+
+			if (ipaddress.isLoopbackAddress() ||
+				ipaddress.isLinkLocalAddress() ||
+				ipaddress.isSiteLocalAddress()) {
```

Comment:
@topolik 
A little beyond my networking knowledge here. Do we also need `isAnyLocalAddress()` to check for wildcard addresses?

Reply:
That should be fine. Looks like `isAnyLocalAddress()` checks `0.0.0.0` that is used to listen on all interfaces. Nothing that could be a target IP to connect to.